### ISSUE TITLE
Ask for the dataset.

### ISSUE THREAD

**Issue by linxin0** [zhichu99/PanoSplatt3R#1]
Impressive work! When will the dataset be uploaded?

Thanks!

**Comment by zhichu99** [comment]
As the training dataset is to large, you can refer to https://github.com/thucz/splatter360/ to get processing code and replica testset. However, the testset of HM3D is still very large(600G+), so we will try to compressing it before uploading.

**Comment by zhichu99** [comment]
Hello! We’ve released our model weights and the HM3D test set now.
